src/aegis_ai/features/data_models.py

```diff
@@ -11,6 +11,11 @@ class AegisFeatureModel(BaseModel):
     Metadata for Aegis features, nested within the main feature model.
     """
 
+    data_quality: Optional[Literal["POOR", "FAIR", "GOOD", "EXCELLENT"]] = Field(
```

Comment:
Why do we use exactly those literals?  What is the benefit over using a continuous value, like we do for `confidence`, `completeness`, and `consistency`?

Also this seems to have no coverage.  I think the value should be checked in [FeatureMetricsEvaluator](https://github.com/RedHatProductSecurity/aegis-ai/blob/97922499985dc1f36a440027df868653a13f0cb2/evals/features/common.py#L52).

Reply:
thx for the comments -  my next PR (which has data_critic tool) will change this to a range and will add coverage in eval.